packages/@o3r/components/src/devkit/inspector/otter-inspector.helpers.spec.ts

```diff
@@ -27,11 +48,12 @@ class MockComponent {
   public translations: Translation = {
     mainKey: 'mainKey'
   };
+
   public analyticsEvents: AnalyticsEvents = {
     mainEvent: MockEvent
   };
 
-  // eslint-disable-next-line @typescript-eslint/naming-convention
+  // eslint-disable-next-line @typescript-eslint/naming-convention -- private field
```

Comment:
private field that happens to be public ? :D

Reply:
We are in a mocked component

Reply:
That doesn't change the fact that the justification for the rule is confusing (the field is clearly not private here)
Maybe instead you can say the the naming is imposed by the implementation in which it is a private field